api/controllers/PokemonController.js

```diff
@@ -73,13 +71,7 @@ module.exports = _.mapValues({
 
   async delete (req, res) {
     const id = req.param('id');
-    let pokemon = await Pokemon.findOne({id});
-    if (!pokemon) {
-      return res.notFound();
-    }
-    if (pokemon.owner !== req.user.name && !req.user.isAdmin) {
-      return res.forbidden();
-    }
+    let pokemon = await Validation.verifyUserIsPokemonOwner({user: req.user, id});
```

Comment:
I mean, it's still technically validation, it just happens to be checking an object against the database rather than itself.

What would you propose as an alternative to keep down boilerplate code? It seems like it would be best to abstract it into a function somewhere, even if we don't call it 'validation'.